compiler/src/iree/compiler/InputConversion/MHLO/MHLOToLinalgOnTensors.cpp

```diff
@@ -394,8 +535,9 @@ void populateMHLOToLinalgOnTensorsConversionPatterns(
   mhlo::populateHloToLinalgConversionPattern(context, typeConverter, &patterns);
   // TODO(#5809): Drop ConcatenateOp lowering in favor of the upstream version
   //              then remove the PatternBenefit here
-  patterns.insert<ConcatenateOpConversion, FftOpConversion>(
-      typeConverter, context, PatternBenefit(1000));
+  patterns.insert<ConcatenateOpConversion, FftOpConversion,
+                  ScatterUpdateConversion>(typeConverter, context,
```

Comment:
Wait, I don't think we want to give this one a higher benefit. I think it should be at like 0 benefit really (only use if nothing else can), and we may even want to emit a warning where it applies that it'll be slow.

Reply:
Good point, done!